### PR TITLE
Add database pair registered service

### PR DESCRIPTION
**Pull request by zhongyi-zhang**
The third step in https://github.com/Azure/open-service-broker-azure/issues/566 for MSSQL DR.

### CODE REVIEW

File: tests/lifecycle/driver_test.go [hunk 1]
```diff
@@ -50,6 +52,9 @@ func TestServices(t *testing.T) {
 }
 
 func TestMain(m *testing.M) {
+	if err := crypto.InitializeGlobalCodec(noop.NewCodec()); err != nil {
+		os.Exit(-1)
+	}
```
Comment (zhongyi-zhang) [inline tests/lifecycle/driver_test.go]: It is something we ever mentioned in https://github.com/Azure/open-service-broker-azure/pull/598. But the PR is still pending.

File: pkg/services/mssqldr/catalog.go [hunk 2]
```diff
@@ -327,5 +327,63 @@ func (m *module) GetCatalog() (service.Catalog, error) {
...
+				Name:        "database",
+				Description: "Azure SQL Server-- database",
+				Free:        false,
+				Stability:   service.StabilityExperimental,
+				Metadata: service.ServicePlanMetadata{
+					DisplayName: "Azure SQL Server-- database",
+				},
+				Schemas: service.PlanSchemas{
+					ServiceInstances: service.InstanceSchemas{
+						ProvisioningParametersSchema: service.InputParametersSchema{
```
Comment (norshtein) [inline pkg/services/mssqldr/catalog.go]: Using a function will make it easier to maintain, just as `m.dbmsPairRegisteredManager.getProvisionParametersSchema()`

Comment (zhongyi-zhang) [reply pkg/services/mssqldr/catalog.go]: Thanks. Fixed.

**Comment by zhongyi-zhang** [comment]
Thanks for the review.

**Pull request merged**